dokku.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -eu
+
+cd /tmp && wget https://github.com/casey/just/releases/download/1.40.0/just-1.40.0-x86_64-unknown-linux-musl.tar.gz && tar -xzf just-1.40.0-x86_64-unknown-linux-musl.tar.gz && chmod 555 just
+sudo mv /tmp/just /usr/bin/
+
+cd /workspaces/opencodelists
+cp dotenv-sample .env
+touch db.sqlite3 # workaround because build-dbs-for-local-development expects a db.sqlite3
+yes Y | just build-dbs-for-local-development nuclear
```

Comment:
thought: `yes` is necessary because `build-dbs-for-local-development` has an interactive user confirmation; not sure if the capitalised `Y` is necessary.

Reply:
thought: The use of `yes` results in failure if you use `set -o pipefail` because there's a broken pipe when the command on the right-hand side completes.

Reply:
thought: We need Python to run the initial setup in `build-dbs-for-local-development`. 

We're currently on Python 3.12 for OpenCodelists.

The default dev container image currently has Python 3.12 available, which is convenient, but more luck than anything else.

To be more confident of always having the correct Python version in future, we'd need to retrieve the required version of Python ourselves.